swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category):
+        cateogry_list = []
+        for item in self.inventory:
+            if item.category == category :
+                cateogry_list.append(item)                
+        return cateogry_list
+
+    def swap_items(self, friend, item_give, item_get):
+        if item_give in self.inventory and item_get in friend.inventory:
```

Comment:
Consider reversing the sense of any logic like this where the else is a simple return so that the return is part of the if. By checking that condition and returning as soon as we detect that some minimum requirements are not met, we convert the if conditional into a "guard clause". Since it returns when run, we don't need to use the else keyword, and we can avoid indenting our main function logic.

This is a very common pattern in production code.